src/transactions/OfferExchange.cpp

```diff
@@ -887,14 +891,53 @@ OfferExchange::crossOfferV10(OfferFrame& sellingWheatOffer,
             TrustFrame::loadTrustLine(accountBID, sheep, db, mDelta);
     }
 
+    Liabilities prevLiabilities;
```

Comment:
you don't need to have `prevLiabilities` defined in this scope, in fact I don't think you need this variable at all.
Also a comment that explains what is going on would help, something in the line of "free up the liabilities of this offer while we cross it"